Formula/z/z80dasm.rb

```diff
@@ -1,7 +1,9 @@
 class Z80dasm < Formula
   desc "Disassembler for the Zilog Z80 microprocessor and compatibles"
-  homepage "https://www.tablix.org/~avian/blog/articles/z80dasm/"
+  # use debian site for now, cannot use debian mirror as it is still with 1.1.6
+  homepage "https://packages.debian.org/sid/z80dasm"
```

Comment:
Can we use this instead like in https://github.com/mistydemeo/tigerbrew/blob/1508070c8a2d42bf915e8597efd7b46ecb72fa6e/Library/Formula/z80dasm.rb#L3?

```suggestion
  homepage "https://web.archive.org/web/20240114192756/https://www.tablix.org/~avian/blog/articles/z80dasm/"
```

Reply:
We disallow archive.org URLs in Homebrew/core.